app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -4,7 +4,8 @@
       #autocomplete-options
         = (event.review_tags | proposal.review_tags)
       = f.text_field :review_tags, { value: proposal.review_tags.join(','), tabindex: '-1' }
-
+      - if program_mode?
+        = hidden_field_tag :program_mode, true
```

Comment:
I saw this being done another way in ratings_controller.  It would be ideal to make them work the same way and have it so program_mode?  would also work as the check for the review_tags submission.